Documentation/network/servicemesh/ingress-to-gateway/http-migration.rst

```diff
@@ -0,0 +1,189 @@
+and ``/`` to the ``productpage`` service.
+
+
+Create Equivalent Gateway Configuration
+=======================================
+
+To create the equivalent Gateway configuration, you must consider the following:
+
+- Entry Point
+
+The entry point is a combination of an IP address and port through which the data plane is accessible to external clients.
+
+.. tabs::
+
+    .. group-tab:: Ingress
+        
+        Every Ingress resource has two implicit entry points -- one for HTTP and the other for HTTPS traffic. 
+        An Ingress controller provides those entry points. Typically, they will either be shared by all Ingress resources, or every Ingress resource will get dedicated entry points.
+
+        .. code-block:: shell-session
+
+            apiVersion: networking.k8s.io/v1
+            kind: Ingress
+            spec:
+              ingressClassName: cilium
+
+    .. group-tab:: Gateway API
+
+        In the Gateway API, entry points must be explicitly defined in a Gateway resource. 
+        For example, if you want the data plane to handle HTTP traffic on port 80, you need to define a listener for that traffic. 
+        Typically, a Gateway implementation provides a dedicated data plane for each Gateway resource.
+
+        .. code-block:: shell-session
+
+            apiVersion: gateway.networking.k8s.io/v1beta1
+            kind: Gateway
+            metadata:
+              name: cilium
+            spec:
+              gatewayClassName: cilium
+              listeners:
+              - name: http
+                port: 80
+                protocol: HTTP
+
+- Routing Rules
+
+When using Ingress or Gateway API, routing rules need to be defined to attach applications to those entry points.
+
+.. tabs::
+
+    .. group-tab:: Ingress
+        
+        The path-based routing rules are configured in the Ingress resource map directly to the routing rules of the HTTPRoute.
+
+        In the Ingress, each hostname has separate routing rules, 
+
+        .. code-block:: shell-session
+
+            apiVersion: networking.k8s.io/v1
+            kind: Ingress
+            [...]
+            rules:
+            - http:
+                paths:
+                - backend:
+                    service:
+                      name: details
+                      port:
+                        number: 9080
+                  path: /details
+                  pathType: Prefix
+                - backend:
+                    service:
+                      name: productpage
+                      port:
+                        number: 9080
+                  path: /
+                  pathType: Prefix
+
+    .. group-tab:: Gateway API
+
+        The routing rules are configured in the HTTPRoute.
+
+        .. code-block:: shell-session
+
+            ---
+            apiVersion: gateway.networking.k8s.io/v1beta1
+            kind: HTTPRoute
+            spec:
+              parentRefs:
+              - name: cilium
+            rules:
```

Comment:
This feels stupid but how about we turn these around, they would read more like a `from -> to` in my head

```yaml
            rules:
            - matches:
              - path:
                  type: PathPrefix
                  value: /
              backendRefs:
              - name: productpage
                port: 9080
```

Reply:
Makes perfect sense. Changed these around.